src/constants/messeges.js

```diff
@@ -0,0 +1,14 @@
+const MESSEGE = Object.freeze({
+  carNamesInput:
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+  roundNumInput: '시도할 횟수는 몇 회인가요?\n',
+  excutionResult: '\n실행 결과',
+  errorCarNumber: '[ERROR] 자동차 2대 이상부터 경주를 할 수 있습니다.',
```

Comment:
```suggestion
const errorString = content => `[ERROR] ${content}`
{
  errorCarNumber: errorString('자동차 2대 이상부터 경주를 할 수 있습니다.'),
```
예를 들면 이런식으로요!
[ERROR]라는 스트링이 오류 메시지에서 반복되니 함수로 선언해서 재사용할 수도 있을 것 같은데 어떠신가요~?